### PR TITLE
Extended LP fee options

### LINKED ISSUE #813: LP fee setting - new options
Highlight the 2 new options for setting liquidity fees and how to choose an option. https://github.com/vegaprotocol/specs/pull/2017/files

### PR DESCRIPTION
**Pull request by candida-d**
Closes #813 

PR #893: Update market proposal templates when new fields can be generated

### CODE REVIEW

File: docs/tutorials/proposals/new-market-proposal.md [hunk 1]
```diff
@@ -168,6 +168,18 @@ The risk model uses the following properties:
...
+
+| Field | Description | Sample value |
+| ----------- | ----------- | ----------- |
+| `liquiditySlaParameters` | Parameters for minimum requirements and measurements | |
+| `priceRange` | Sets the percentage move up and down from the mid price that LPs must be within to count towards their commitment | 0.1 | 
+| `commitmentMinTimeFraction` | The minimum fraction of time that LPs must spend on the book and within the price range | 0.1 | 
+| `performanceHysteresisEpochs` | Sets the number of epochs over which past performance will continue to affect rewards. | 10 | 
+| `slaCompetitionFactor` | Sets the maximum fraction of their accrued fees an LP that meets the SLA will lose to liquidity providers that achieved a higher SLA performance than them. | 0.2 |
+| `liquidityFeeSettings` | Optional setting for how the liquidity fee factor is determined. See [liquidity fees](../../concepts/liquidity/rewards-penalties.md#determining-the-liquidity-fee-percentage) for more. | `METHOD_MARGINAL_COST` (default) `METHOD_CONSTANT`, `METHOD_WEIGHTED_AVERAGE` |
```
Comment (candida-d) [inline docs/tutorials/proposals/new-market-proposal.md:181]: Add detail about `feeConstant` sub-field if constant is chosen as a "string"

File: docs/concepts/liquidity/rewards-penalties.md [hunk 2]
```diff
@@ -60,12 +60,30 @@ Read more: How SLA performance is calculated (spec when out of CE branch)
...
-4. Once the fee for the market is set, all liquidity orders charge that fee, regardless of whether the provider's submitted fee was higher or lower, and whatever the proposed fee factor.
-5. This fee percentage can change. Once the market passes governance and its opening auction begins, a clock starts ticking. The protocol calculates the target stake, and the fee is continuously re-evaluated. Liquidity providers amending or cancelling their commitments will also lead to the fee factor changing.
+There are three options: 
+
+1. **Constant fee**
+
+The market's liquidity fee is provided in the market proposal and this percentage doesn't vary. The fee percentage can be changed in an update market proposal.
```
Comment (davidsiska-vega) [inline docs/concepts/liquidity/rewards-penalties.md:69]: And the method can be changed in an update market proposal - I am not sure that needs to be stressed.

File: docs/concepts/liquidity/rewards-penalties.md [hunk 3]
```diff
@@ -60,12 +60,30 @@ Read more: How SLA performance is calculated (spec when out of CE branch)
...
+There are three options: 
+
+1. **Constant fee**
+
+The market's liquidity fee is provided in the market proposal and this percentage doesn't vary. The fee percentage can be changed in an update market proposal. 
+
+
+2. **Weighted by each provider's supplied liquidity**
+
+Each liquidity provider submits their desired liquidity fee factor in the liquidity commitment transaction. It’s a number between 0 and 1. Each fee factor has a weight assigned to it based on the supplied liquidity from each provider. The highest weighted fee factor is then used as the liquidity fee factor. The final fee factor is converted into a percentage.
```
Comment (davidsiska-vega) [inline docs/concepts/liquidity/rewards-penalties.md:74]: @Jiajia-Cui I thought it's the weighted average, not the LP with the highest weight. Is that correct?

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
Left a comment and a question for Jiajia.

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**